judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/submission/programming/ContestSubmissionResource.java

```diff
@@ -144,6 +149,11 @@ public ContestSubmissionsResponse getSubmissions(
         userJids = submissions.getPage().stream().map(Submission::getUserJid).collect(Collectors.toSet());
         if (canSupervise) {
             userJids.addAll(contestantStore.getApprovedContestantJids(contestJid));
+            userJids.addAll(supervisorStore
```

Comment:
Hmm. Can we just implement `getAllSupervisorJids()`  so that it's simillar to the above call for contestants? Only JIDs are required anyway in this use case.